ClassLibrary/clsStaff.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
 
 namespace ClassLibrary
 {
     public class clsStaff
     {
-        private int pStaffID;
+        private Int32 pStaffID;
```

Comment:
These two staff Id fields don't have the same data type, is this intentional?

Reply:
Mistake.